src/regiment/mod.rs

```diff
@@ -40,8 +40,8 @@ impl Regiment {
     ///
     /// (usize) nb_models: The number of models in the regiment
     ///
-    /// Option<usize> regiment_health_point: The number of health points that left to the regiment.
-    /// If None, the default value will be (the number of model * the health_point stats of the model)
+    /// Option<usize> regiment_health_points: The number of health points that left to the regiment.
+    /// If None, the default value will be (the number of model * the health_points stats of the model)
```

Comment:
Why add Option here ?
The regiment health point can't be just `nb_rows * nb_cols * model_health_points` or `nb_model * model_health_points` and reduce during the game ?

Reply:
we talked about it during a meeting. As there are some models that have more than one HP.
without this regiment_health_point  we can't take account when a unit lost 1 HP but is not dead. The code is already implemented and was validated in a previous PR :|